energy-integration/fees-collector/src/external_sc_interactions/pair.rs

```diff
@@ -0,0 +1,36 @@
+multiversx_sc::imports!();
+
+mod pair_proxy {
+    multiversx_sc::imports!();
+
+    #[multiversx_sc::proxy]
+    pub trait PairProxy {
+        #[payable("*")]
+        #[endpoint(swapTokensFixedInput)]
+        fn swap_tokens_fixed_input(
```

Comment:
Why not integrate the actual Pair contract and write just a proxy? This way you can make a complete integration test.